jobs/delete-version.groovy

```diff
@@ -15,7 +15,7 @@ new Setup(steps
 // We run this job once every few minutes; 100 builds covers about
 // 30 minutes.  Let's keep at least a days' around, for debugging.
 ).resetNumBuildsToKeep(
-   5000,
+   9000,
```

Comment:
so based on this 100 builds a day, assuming that we have about an 11 hr work day covering east to west coast (not taking into account folks working interesting work hours)...  If we have roughly 200 builds an 1 hour, 9000 builds to keep will roughly translate to 4 work days worth logs?  That seems good to me.